region/norfair/crocomire/Post Crocomire Jump Room.json

```diff
@@ -318,6 +343,51 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "link": [1, 6],
+      "name": "G-Mode Morph Spring Ball",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphSpringBall"
+      ]
+    },
+    {
+      "link": [1, 6],
+      "name": "G-Mode Morph Diagonal IBJ",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphDiagonalBombJump",
+        "canBeVeryPatient"
+      ]
+    },
+    {
+      "link": [1, 6],
+      "name": "G-Mode Morph Acid Dive IBJ",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "Gravity",
+        "h_canArtificialMorphBombHorizontally",
+        "h_canArtificialMorphIBJ",
+        "canSuitlessLavaDive",
+        {"acidFrames": 175}
+      ],
+      "note": "Bomb boost horizontally into the acid, then IBJ onto the solid platforms."
```

Comment:
Starting a double in acid seems a bit scary. I was hesitant on the ibj in general, but it can be done tankless with varia or with 1 tank, so I think it's worth keeping in.